playground/.eslintrc.cjs

```diff
@@ -0,0 +1,23 @@
+module.exports = {
```

Comment:
@gabrielse, I've cherry-picked your commit for #18. Let's talk about the Playground here. I know that @brennj also had some ideas in mind for it, so please coordinate among yourselves, okay? :)